src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -9,6 +9,12 @@ define(function(require) {
 
 		// Defines API requests not included in the SDK
 		requests: {
+			'phonebook.lookupNumbers': {
+				apiRoot: monster.config.api.phonebook,
+				url: 'lnp/lookup',
+				verb: 'POST',
+				generateError: false
```

Comment:
Is this parameter needed?

Reply:
In this case, yes. A custom message is shown if the request fails, to let the user know that the Phonebook service may be unavailable.